content/docs/csidriver/release/powerflex.md

```diff
@@ -6,19 +6,17 @@ description: Release notes for PowerFlex CSI driver
 ## Release Notes - CSI PowerFlex v2.8.0
 
 
+
 ### New Features/Changes
 
 - [#724 - [FEATURE]: CSM support for Openshift 4.13](https://github.com/dell/csm/issues/724)
 - [#763 - [FEATURE]: CSI-PowerFlex 4.0 NFS support](https://github.com/dell/csm/issues/763)
 - [#876 - [FEATURE]: CSI 1.5 spec support -StorageCapacityTracking](https://github.com/dell/csm/issues/876)
-- [#926 - [FEATURE]: Set up golangci-lint for all CSM repositories](https://github.com/dell/csm/issues/926)
```

Comment:
This also should not get removed.

Reply:
@VamsiSiddu-7 Milestone is not added for this issue, if you want this to be added here, please add milestone from your end.

Reply:
added the milestone